README.md

```diff
@@ -39,6 +39,8 @@ Source releases and Windows binaries for OpenAL Soft are
 available at its [homepage](https://openal-soft.org/).
 Instructions are also provided there.
 
+Users on Windows 8 and above may need to add certain DirectSound registry keys for the DLL to work: [dsound.zip](https://github.com/kcat/dsoal/files/10252285/dsound.zip)
```

Comment:
I'm not sure what the link is supposed to do to clarify the issue or show what to do.

Reply:
How material is this risk actually? In the sense of do we know how many games actually use fixed paths to address dsound.dll and not always prefer a dsound.dll file in the game folder itself.
I know this can happen when no file available locally (so not in the game folder, but e.g., in the Windows System folder), but regular Windows behavior is to always look in the root folder first and only then check other locations for required files (regular FilePath behavior that is pretty common amongst most operating systems).

Thus far I've always just dropped the 3 files from the binary distributions in the game folder and everything works as planned. Didn't even make any changes to the alsoft configuration file.

Reply:
As I pointed out in https://github.com/kcat/dsoal/issues/34#issuecomment-1417967253, this is eventually only about some games using `CoCreateInstance` instead of `LoadLibrary` to load directsound.